### ISSUE TITLE
'Lark' and 'Moon' are too similar.

### ISSUE THREAD

**Issue by yunsung-soh** [hsbijarniya/colorfilter_generator#1]
Hello,
'Lark' and 'Moon' look too similar. Perhaps there is a problem with 'Lark'.

**Comment by georgerappel** [comment]
Good point, looking at the code the single difference between the two is 0.02 (2%) change in brightness, which doesn't align with what the comment explaining the filter expects it to do.

https://github.com/hsbijarniya/colorfilter_generator/blob/fd08bdf5afbc08241d001cd17b6826e58cda8831/lib/presets.dart#L47-L65
